modules/core/src/test/scala/com.snowplowanalytics.snowplow.bigquery/processing/ProcessingSpec.scala

```diff
@@ -621,19 +632,117 @@ class ProcessingSpec extends Specification with CatsEffect {
   def e12       = e12Base(legacyColumns = false)
   def e12Legacy = e12Base(legacyColumns = true)
 
+  def e13 = {
```

Comment:
The title of this test, taken from above, is

```
Use legacy columns for all fields when legacyColumnMode is enabled $e13
```

Do you think that is a good description of what is actually tested in the test implementation?  Looks like you are just testing that it sets various atomic fields.  It's a nice check... but the loader should pass this test whether or not `legacyColumnMode` is enabled.